Test

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+set -e
+
+trap 'cleanup; echo >&2 "FAILED!"' 0
+cleanup() { progress "Cleaning up"; kill $(jobs -p) >/dev/null ; }
+clean_exit() {
+    cleanup
+    trap '' 0
+    error_code=$(shift)
+    echo >&2 "$@"
+    exit $error_code
+}
+
+type pytest > /dev/null \
+    || clean_exit 1 -n \
+            'Please install pytest for python2 via pip.\n' \
+            'On ubuntu xenial run:: `pip2 install pip && pip2 install --user pytest`'
+
```

Comment:
I simply got a `pytest command not found error` instead of the above mentioned message. Not sure why.